make error trying to build Intel Aero Realsense

Hello - 

I am following the steps in the Intel Aero Wiki to install librealsense but run into the following error during the `make` step. 
https://github.com/intel-aero/meta-intel-aero/wiki/90-(References)-OS-user-Installation#intel-realsense-sdk

I am using Ubuntu 16.04LTS and kernel 
```
uname -r
4.4.0-1079-aws
```


```
 sudo apt-get -y install git libssl-dev libusb-1.0-0-dev pkg-config libgtk-3-dev libglfw3-dev cmake
 git clone -b legacy --single-branch https://github.com/IntelRealSense/librealsense.git
 cd librealsense
 mkdir build && cd build
 cmake ../ 
 make     -->  fails as shown below
```

Here is the output from make 

```
make
Scanning dependencies of target realsense
[  1%] Building CXX object CMakeFiles/realsense.dir/src/archive.cpp.o
[  3%] Building CXX object CMakeFiles/realsense.dir/src/context.cpp.o
[  5%] Building CXX object CMakeFiles/realsense.dir/src/device.cpp.o
[  7%] Building CXX object CMakeFiles/realsense.dir/src/ds-device.cpp.o
[  9%] Building CXX object CMakeFiles/realsense.dir/src/ds-private.cpp.o
[ 11%] Building CXX object CMakeFiles/realsense.dir/src/f200.cpp.o
[ 13%] Building CXX object CMakeFiles/realsense.dir/src/hw-monitor.cpp.o
[ 15%] Building CXX object CMakeFiles/realsense.dir/src/image.cpp.o
[ 17%] Building CXX object CMakeFiles/realsense.dir/src/ivcam-private.cpp.o
[ 19%] Building CXX object CMakeFiles/realsense.dir/src/ivcam-device.cpp.o
[ 21%] Building CXX object CMakeFiles/realsense.dir/src/log.cpp.o
[ 23%] Building CXX object CMakeFiles/realsense.dir/src/motion-module.cpp.o
[ 25%] Building CXX object CMakeFiles/realsense.dir/src/r200.cpp.o
[ 26%] Building CXX object CMakeFiles/realsense.dir/src/rs.cpp.o
[ 28%] Building CXX object CMakeFiles/realsense.dir/src/sr300.cpp.o
[ 30%] Building CXX object CMakeFiles/realsense.dir/src/stream.cpp.o
[ 32%] Building CXX object CMakeFiles/realsense.dir/src/sync.cpp.o
[ 34%] Building CXX object CMakeFiles/realsense.dir/src/timestamps.cpp.o
[ 36%] Building CXX object CMakeFiles/realsense.dir/src/types.cpp.o
[ 38%] Building CXX object CMakeFiles/realsense.dir/src/uvc-libuvc.cpp.o
[ 40%] Building CXX object CMakeFiles/realsense.dir/src/uvc-v4l2.cpp.o
/home/ubuntu/librealsense/src/uvc-v4l2.cpp: In constructor ‘rsimpl::uvc::subdevice::subdevice(const string&)’:
/home/ubuntu/librealsense/src/uvc-v4l2.cpp:158:39: error: ‘V4L2_CAP_META_CAPTURE’ was not declared in this scope
                 if((cap.device_caps & V4L2_CAP_META_CAPTURE)){
                                       ^
CMakeFiles/realsense.dir/build.make:542: recipe for target 'CMakeFiles/realsense.dir/src/uvc-v4l2.cpp.o' failed
make[2]: *** [CMakeFiles/realsense.dir/src/uvc-v4l2.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/realsense.dir/all' failed
make[1]: *** [CMakeFiles/realsense.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
```

The variable `V4L2_CAP_META_CAPTURE` is not defined anywhere in the source code after the git clone. 

Am I missing a command or 2 ?

@dorodnic @sgorsten @aangerma 

Thanks

--- a/src/uvc-v4l2.cpp
+++ b/src/uvc-v4l2.cpp
@@ -41,6 +41,10 @@
 
 #pragma GCC diagnostic ignored "-Woverflow"
 
+#ifndef V4L2_CAP_META_CAPTURE
+#define V4L2_CAP_META_CAPTURE    0x00800000  /* Specified in kernel header v4.16, required for back-compat */
+#endif // V4L2_CAP_META_CAPTURE
+
 namespace rsimpl
 {
     namespace uvc

